EST.MIT.InvoiceImporter.Function/Functions/UploadFunctions.cs

```diff
@@ -39,4 +41,27 @@ public UploadFunctions(IAzureTableService azureTableService)
             return new StatusCodeResult((int)HttpStatusCode.InternalServerError);
         }
     }
+
+    [FunctionName("GetUploadedFile")]
+    public async Task<IActionResult> GetUploadedFile(
+      [HttpTrigger(AuthorizationLevel.Function, "get", Route = "UploadedFile/{Ref}")] HttpRequest req,
+      string Ref,
+      ILogger log)
+    {
+        log.LogInformation($"[GetUploadedFile] Received request for Ref: {Ref}");
+
+        var importRequest = await _azureTableService.GetUserImportRequestsByImportRequestIdAsync(Ref);
```

Comment:
Wondering if any chain of activity might end up with two TableService rows with the same RequestId existing?

Reply:
A future ticket will flesh out the implementation of uploading files, and we can address how to generate unique IDs then.